aip/general/0127.md

```diff
@@ -81,7 +81,8 @@ message CreateBookRequest {
 - The `body` key defines which single top-level field in the request will be
   sent as the HTTP body. If the body is `*`, then this indicates that the
   request object itself is the HTTP body. The request body is encoded as JSON
-  as defined by protocol buffers' canonical [JSON encoding][].
+  as defined by protocol buffers' canonical [JSON encoding][]. Fields encoded
+  in the URI
```

Comment:
Incomplete sentence.

Reply:
Whoops - started putting it there, then moved it. Fixed now.